src/Polymath.ts

```diff
@@ -431,6 +435,32 @@ export class Polymath {
     return await procedure.prepare();
   };
 
+  public forceTransfer = async (args: {
+    securityTokenId: string;
+    value: BigNumber;
+    from: string;
+    to: string;
+    log: string;
```

Comment:
The `log` parameter (ambiguously named so by the core team) is actually a reason message for the transfer. We should call it `reason` to be more explicit and make it optional as well (defaulting to empty string as in the LowLevel implementation)